routes/api.php

```diff
@@ -195,4 +195,14 @@
     'updatePaymentMethod',
 ]);
 
+$router->post('/users/{key}/email', [
```

Comment:
@rwhite27 make it explicit change-email

Reset-key

Reply:
should it be then **/users/{key}/reset-key**?